src/index/views/4-Dashboard.vue

```diff
@@ -218,6 +197,7 @@ export default {
         this.timeRechargeMana = deltaTimeToString(
           ((balance - this.mana) * FIVE_DAYS) / balance
         );
+        this.manaPercent = Math.floor(this.mana / balance * 100) || 1;
```

Comment:
That's the way I did it on burn koin. The tooltip there shows "<1%" instead of time to recharge. By setting manaPercent to 1, you still see a small amount or purple in the bottom of the orb instead of a completely empty orb.

I'm not attached to it. I can remove the minimum.

Reply:
done